walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -81,24 +78,26 @@
             if (nextTimeRange.end()<=currentTimeRange.end())
             {
                 newEvents.set(i+1, new Event("hi", currentTimeRange, new ArrayList<>())); 
+                nextTimeRange = currentTimeRange; 
             }
             else 
             {
-                newEvents.set(i+1, new Event ("hello", TimeRange.fromStartDuration(currentTimeRange.start(), nextTimeRange.end()-currentTimeRange.start()), new ArrayList<>()));
+                newEvents.set(i+1, new Event("hello", TimeRange.fromStartDuration(currentTimeRange.start(), nextTimeRange.end()-currentTimeRange.start()), 
```

Comment:
Can we store `currentTimeRange.start()` as a variable because it keeps getting used? Same for other places that do this (ie `nextTimeRange.end()`)